scripts/products.json

```diff
@@ -1,66 +1,74 @@
-      "image": "./img/BottleShots_Rose.png"
-    },
-    {
-      "id": "4",
-      "name": "Red Wine 4",
-      "type": "Red Wine",
-      "price": 13.99,
-      "description": "This is Red Wine 4",
-      "image": "./img/BottleShots_Rose.png"
-    },
-    {
-      "id": "5",
-      "name": "White Wine 5",
-      "type": "White Wine",
-      "price": 14.99,
-      "description": "This is White Wine 5",
-      "image": "./img/BottleShots_Rose.png"
-    },
-    {
-      "id": "6",
-      "name": "White Wine 6",
-      "type": "White Wine",
-      "price": 15.99,
-      "description": "This is White Wine 6",
-      "image": "./img/BottleShots_Rose.png"
-    },
-    {
-      "id": "7",
-      "name": "White Wine 7",
-      "type": "White Wine",
-      "price": 16.99,
-      "description": "This is White Wine 7",
-      "image": "./img/BottleShots_Rose.png"
-    },
-    {
-      "id": "8",
-      "name": "White Wine 8",
-      "type": "White Wine",
-      "price": 17.99,
-      "description": "This is White Wine 8",
-      "image": "./img/BottleShots_Rose.png"
-    }
-  ]
+  {
+    "id": "1",
+    "name": "Red Wine 1",
+    "type": "Red Wine",
+    "price": 10.99,
+    "quantity": "20",
+    "description": " Kenwood Vineyards Sonoma Coast Merlot 2018",
+    "image": "./img/BottleShot_Yoakim_Zin.png"
+  },
+  {
+    "id": "2",
+    "name": "Red Wine 2",
+    "type": "Red Wine",
+    "price": 10.99,
+    "quantity": "20",
+    "description": " Kenwood Vineyards Sonoma Coast Merlot 2018",
+    "image": "./img/BottleShot_Yoakim_Zin.png"
+  },
+  {
+    "id": "3",
+    "name": "Red Wine 3",
+    "type": "Red Wine",
+    "price": 10.99,
+    "quantity": "20",
+    "description": " Kenwood Vineyards Sonoma Coast Merlot 2018",
+    "image": "./img/BottleShot_Yoakim_Zin.png"
+  },
+  {
+    "id": "4",
+    "name": "Red Wine 4",
+    "type": "Red Wine",
+    "price": 10.99,
+    "quantity": "20",
+    "description": " Kenwood Vineyards Sonoma Coast Merlot 2018",
+    "image": "./img/BottleShot_Yoakim_Zin.png"
+  },
+  {
+    "id": "5",
+    "name": "White Wine 5",
+    "type": "White Wine",
+    "price": 10.99,
+    "quantity": "20",
+    "description": " Kenwood Vineyards Sonoma Coast Merlot 2018",
+    "image": "./img/BottleShot_Yoakim_Zin.png"
+  },
+  {
+    "id": "6",
+    "name": "White Wine 6",
+    "type": "White Wine",
+    "price": 10.99,
+    "quantity": "20",
+    "description": " Kenwood Vineyards Sonoma Coast Merlot 2018",
+    "image": "./img/BottleShot_Yoakim_Zin.png"
+  },
+  {
+    "id": "7",
+    "name": "White Wine 7",
+    "type": "White Wine",
+    "price": 10.99,
+    "quantity": "20",
+    "description": " Sonoma Coast Merlot 2018",
+    "image": "./img/BottleShot_Yoakim_Zin.png"
+  },
+  {
+    "id": "8",
+    "name": "White Wine 8",
+    "type": "White Wine",
+    "price": 10.99,
+    "quantity": "20",
+    "description": " Kenwood Vineyards Sonoma Coast Merlot 2018",
+    "image": "./img/BottleShot_Yoakim_Zin.png"
+  }
+]
```

Comment:
We need to actually fill this json file with different data eventually.
Also quantity field is not needed really so we can remove that.